plugins/modules/vmware_vm_info.py

```diff
@@ -273,6 +275,16 @@ def get_all_virtual_machines(self):
             if self.module.params.get('show_attribute'):
                 vm_attributes = self.get_vm_attributes(vm)
 
+            vm_folder = ""
```

Comment:
Can we re-use existing [`get_vm_path`](https://github.com/ansible/ansible/blob/833970483100bfe89123a5718606234115921aec/lib/ansible/module_utils/vmware.py#L1089) function here?

Reply:
I'm think that there's no problem in that. 
I'm didn't know about that function before.

Reply:
Ok. No problem. Please ping me once you are done with the change.

Reply:
Changes done @Akasurde .